components/date-picker/utils.ts

```diff
@@ -2,13 +2,18 @@ import * as moment from 'moment';
 
 export function formatDate(
   value: moment.Moment | undefined | null,
-  format: string | string[],
+  format: string | string[] | ((moment: moment.Moment | undefined) => string),
 ): string {
   if (!value) {
     return '';
   }
   if (Array.isArray(format)) {
     format = format[0];
   }
+
+  if (typeof format === 'function') {
+    format = format(value);
+  }
```

Comment:
You should start with http://github.com/react-component/calendar

Reply:
👌

Reply:
[#599](https://github.com/react-component/calendar/pull/599)